packages/snjs/mocha/features.test.js

```diff
@@ -226,4 +230,29 @@ describe('features', () => {
       expect(await application.getSensitiveSetting(SettingName.ExtensionKey)).to.equal(true);
     });
   });
+
+  describe('full sync completed observer', () => {
+    it('should update extension key user setting when local data is loaded and user is signed in', async () => {
+      expect(await application.getSensitiveSetting(SettingName.ExtensionKey)).to.equal(false);
+      const extensionKey = Uuid.GenerateUuidSynchronously().split('-').join('');
+      application = await Factory.signOutApplicationAndReturnNew(application);
+      sinon.stub(application.itemManager, 'getItems').callsFake(() => {
+        return [
+          {
+            safeContent: {
```

Comment:
safeContent is a getter, not an actual property.

Reply:
Yup I know but this is just a mock function, can we mock a getter in a different way?

Reply:
Hmm, I suppose it's just a weird way to write the test. You're relying too much on the fact that the client is calling safeContent, but its use is optional in the code.